README.md

```diff
@@ -1,11 +1,66 @@
-# jupiter-repo-dependencies
+# JupiterOne Repository Relationship Generator
 
-- Put the repos you want to run the script on in the `repos` folder.
-- Use `yarn deps` to run the script.
+A script that generates graphql relationships between a repository and its dependencies.
+This script will look for dependencies within a repository's `package.json` file as well as the `dependencies.yaml`file if the repository has a `deploy` directory.
 
-## Package Scope Input
+This script uses the JupiterOne graphql graph database.
```

Comment:
Strictly speaking, the JupiterOne graph database is distinct from the use of GraphQL in the API interfaces to the JupiterOne system. I think this sentence can be dropped.